Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
IMHO flipping the elements inside the method is also a BC break. `configureOptions` should be the right place to flip the elements, because we pass the options to symfony who can handle the flipped elements correctly.